source/boundary_traction/interface.cc

```diff
@@ -59,8 +59,375 @@ namespace aspect
     void
     Interface<dim>::parse_parameters (dealii::ParameterHandler &)
     {}
+// ------------------------------ Manager -----------------------------
+// -------------------------------- Deal with registering boundary_traction models and automating
+// -------------------------------- their setup and selection at run time
+// just copy and paste mangaer class from boundary-velocity/interface.cc from line 66 -428  {need to change the descriptions....}
```

Comment:
This type of documentation should be in the header file following the same format as the other documentation strings there.